client/src/partials/cash/extra_payment/extra_payment.html

```diff
@@ -1,163 +1,212 @@
+                <thead>
+                  <tr>
+                    <th width="7%">{{ "CASH.EXTRA.SALE_NO" | translate }}</th>
+                    <th>{{ "CASH.EXTRA.SALE_UUID" | translate }}</th>
+                    <th>{{ "CASH.EXTRA.SALE_CLIENT" | translate }}</th>
+                    <th>{{ "CASH.EXTRA.SALE_AMOUNT_TO_PAY" | translate }}</th>
+                    <th>{{ "CASH.EXTRA.SALE_AMOUNT_PAYED" | translate }}</th>
+                    <th width="5%">{{ "CASH.EXTRA.SALE_CURRENCY" | translate }}</th>
+                    <th width="10%">{{ "CASH.EXTRA.ACTION" | translate }}</th>
+                  </tr>
+                </thead>
+                <tbody>
+                  <tr 
+                  ng-if="sale.transaction" 
+                  ng-repeat="sale in saleDataUnique | filter : session.search">
+                    <td>{{ sale.transaction }}</td>
+                    <td>{{ sale.inv_po_id }}</td>
+                    <td>
+                      <span ng-if="sale.debitor_first">{{ sale.debitor_first }}</span> 
+                      <span ng-if="sale.debitor_middle">{{ sale.debitor_middle }}</span> 
+                      <span ng-if="sale.debitor_last">{{ sale.debitor_last }}</span> 
+                    </td>
+                    <td><span ng-bind-html="sale.balance | intlcurrency:sale.currency"></span></td>
+                    <td>
+                      <input 
+                      ng-model="sale.cost"
+                      type="number" 
+                      step="0.1"
+                      class="form-invoice" 
+                      placeholder="...">
+                    </td>
+                    <td>
+                      <select 
+                      ng-model="sale.currency" 
+                      style="padding-top:1px;padding-bottom:1px;" 
+                      ng-options="currency.id as currency.symbol for currency in session.currencies.data"
+                      ng-change="setCurrency(sale)">
+                      </select>
+                    </td>
+                    <td>
+                      <bouton  
+                      class="btn btn-success" 
+                      style="padding-top:1px;padding-bottom:1px;" 
+                      ng-click="submit(sale)" 
+                      ng-disabled="!sale.currency">
+                        {{ "CASH.EXTRA.SALE_SUBMIT" | translate }}
+                        <i class="glyphicon glyphicon-check"></i>
+                      </bouton>
+                    </td>
+                  </tr>
+                </tbody>
+              </table>
+            </div>
+          </div>
+        </div>
+      </div>
+    </main>
 
-<script type="text/ng-template" id="AccountList.html">
-  <a>
-    <span bind-html-unsafe="match.label | typeaheadHighlight:query"></span>
-    <span><b>[{{ match.model.account_number }}]</b></span>
-  </a>
-</script>
+    <script type="text/ng-template" id="CreditorList.html">
```

Comment:
It would be good to move this template into an html file.  Like "CreditorList.html".